cmd/agent/app/reporter/tchannel/reporter_test.go

```diff
@@ -36,16 +36,18 @@ import (
 	"github.com/uber/jaeger/thrift-gen/zipkincore"
 )
 
-func initRequirements(t *testing.T) (*metrics.LocalFactory, *testutils.MockTCollector, Reporter) {
+func initRequirements(t *testing.T) (*metrics.LocalFactory, *testutils.MockTCollector, *Reporter) {
 	metricsFactory, collector := testutils.InitMockCollector(t)
-	reporter := NewTChannelReporter("jaeger-collector", collector.Channel, metricsFactory, zap.NewNop())
+	reporter := New("jaeger-collector", collector.Channel, nil, metricsFactory, zap.NewNop())
 	return metricsFactory, collector, reporter
 }
 
 func TestZipkinTChannelReporterSuccess(t *testing.T) {
 	metricsFactory, collector, reporter := initRequirements(t)
 	defer collector.Close()
 
+	assert.NotNil(t, reporter.Channel())
```

Comment:
the test does not depend on this method returning non-nil, so it's an assert, not require. 
In fact, it's not needed here, I'll remove it.